src/Exceptions/UnresolvedLocation.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Locations\Exceptions;
+
+use Exception;
+use Tipoff\Locations\Models\Market;
+
+class UnresolvedLocation extends Exception
+{
+    protected $market;
+
+    public function __construct(Market $market)
+    {
+        parent::__construct("Could not resolve location for { $market->toJson() }");
+
+        $this->market = $market;
+    }
+
+    public function render()
+    {
+        return view('website.markets.select', [
+            'market' => $this->market,
+            'html' => null,
+            'seotitle' => $this->market->title,
+            'seodescription' => "{ $this->market->title } has { $this->market->rooms->count() } different escape rooms and offers private escape games for groups & parties. Book your escape room today!",
+            'ogtitle' => $this->market->title,
+            'ogdescription' => "{ $this->market->title } has { $this->market->rooms->count() } different escape rooms and offers private escape games for groups & parties. Book your escape room today!",
+            'canonical' => "https://thegreatescaperoom.com{ $this->market->bookings_path }",
+            'image' => $this->market->image_id === null ? null : $this->market->image,
+            'ogimage' => $this->market->ogimage_id === null ? url('public/img/ogimage.png') : $this->market->ogimage,
+        ]);
```

Comment:
@drewroberts  Any thoughts on how you want to handle something like this?  Pulling in just `website.markets.select` view assumes there is a `website.layout` view, etc.  Should it only render if the view exists?  Maybe pushing the view name into its config as well?  (Also need to deal the default image and canonical url root)